content/en/about/contributing/style/_index.md

```diff
@@ -469,5 +446,73 @@ to re-enable the `no-inline-html` check.
 {{% /alert %}}
 
 {{% alert title="Important" color="warning" %}} Always use the tag that is
-providing the proper semantic: e.g. for a list use `<ul>` and `<li>`, not `<br />`.
+providing the proper semantic: e.g. for a list use `<ul>` and `<li>`,
+not `<br />`.
 {{% /alert %}}
+
+### Dealing with duplicate content
+
+When the same procedure needs to be described for multiple platforms, or when
+the same code has to be exemplified for multiple languages, it is possible
+that the automatic checkers will flag these as duplicates.
+
+For example, describing the following procedure will result in duplicates
+being reported:
+
+<!--
+// jscpd:ignore-start
+-->
+<!-- markdownlint-disable line-length -->
```

Comment:
Branch should always be up to date before being able to merge, if you merge from there you just need to `git pull` in your fork. Otherwise you can use `git rebase` or `git merge`.
The final tests will always run against an up to date branch.

Reply:
I was only rebasing before creating new branches, thought that bringing in many commits would make the review much harder. Done, removed all exceptions in the About section.

Reply:
If the commits are properly rebased they don't appear in the PR, as the diff is made against the main branch.